WordPressAuthenticator/Unified Auth/View Related/Login/MagicLinkRequestedViewController.xib

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
<img width="329" alt="Screenshot 2022-08-11 at 9 50 00 AM" src="https://user-images.githubusercontent.com/524475/184064236-3b697381-0318-4cda-a8b1-bf9f5435dad3.png">
IMHO, it might be helpful to fix these constraints to make changes to this file in future.